testsuite/features/build_validation/reposync/srv_sync_all_products.feature

```diff
@@ -373,6 +373,46 @@ Feature: Synchronize products in the products page of the Setup Wizard
     When I use spacewalk-common-channel to add channel "sle-micro-5.5-devel-uyuni-client" with arch "x86_64"
     And I wait until the channel "sle-micro-5.5-devel-uyuni-client-x86_64" has been synced
 
+@susemanager
+@slmicro60_minion
+  Scenario: Add SUSE Linux Micro 6.0
+    Given I am authorized for the "Admin" section
+    When I follow the left menu "Admin > Setup Wizard > Products"
+    And I wait until I do not see "currently running" text
+    And I wait until I do not see "Loading" text
+    And I enter "SUSE Linux Micro 6.0" as the filtered product description
+    And I select "SUSE Linux Micro 6.0 x86_64" as a product
+    Then I should see the "SUSE Linux Micro 6.0 x86_64" selected
+    When I open the sub-list of the product "SUSE Linux Micro 6.0 x86_64"
+    And I select "SUSE Manager Client Tools for SL Micro 6 x86_64" as a product
+    Then I should see the "SUSE Manager Client Tools for SL Micro 6 x86_64" selected
+    When I click the Add Product button
+    And I wait until I see "Selected channels/products were scheduled successfully for syncing." text
+    And I wait until I see "SUSE Linux Micro 6.0 x86_64" product has been added
+    And I wait until all synchronized channels for "sl-micro-6.0" have finished
+
+@uyuni
+@slmicro60_minion
+  Scenario: Add SUSE Linux Micro 6.0
+    Given I am authorized for the "Admin" section
+    When I follow the left menu "Admin > Setup Wizard > Products"
+    And I wait until I do not see "currently running" text
+    And I wait until I do not see "Loading" text
+    And I enter "SUSE Linux Micro 6.0" as the filtered product description
+    And I select "SUSE Linux Micro 6.0 x86_64" as a product
+    Then I should see the "SUSE Linux Micro 6.0 x86_64" selected
+    When I click the Add Product button
+    And I wait until I see "Selected channels/products were scheduled successfully for syncing." text
+    And I wait until I see "SUSE Linux Micro 6.0 x86_64" product has been added
+    And I wait until all synchronized channels for "sl-micro-6.0" have finished
+
+# TODO Enable it when it's available here: https://github.com/uyuni-project/uyuni/blob/600ccd4535b7696d16d990d5835d3711f65feed1/utils/spacewalk-common-channels.ini#L1981-L2001
+# @uyuni
+# @slmicro60_minion
+#   Scenario: Add SUSE Linux Enterprise Micro 6.0 Uyuni Client tools
+#     When I use spacewalk-common-channel to add channel "sl-micro-6.0-devel-uyuni-client" with arch "x86_64"
+#     And I wait until the channel "sl-micro-6.0-devel-uyuni-client-x86_64" has been synced
```

Comment:
This is available now, too, and can be uncommented.
See https://github.com/uyuni-project/uyuni/commit/379ae022872cb7be5b25611bbd56d40a02b400d4

Reply:
checked and corrected, will test as soon as the port is also adjusted